spring-rabbit/src/test/java/org/springframework/amqp/rabbit/log4j/AmqpAppenderIntegrationTests.java

```diff
@@ -67,9 +66,10 @@
 
 	private SimpleMessageListenerContainer listenerContainer;
 
+	@SuppressWarnings("deprecation") // SF 4.2.1
 	@Before
 	public void setUp() throws Exception {
-		Log4jConfigurer.initLogging("classpath:log4j-amqp.properties");
+		org.springframework.util.Log4jConfigurer.initLogging("classpath:log4j-amqp.properties");
```

Comment:
Cool! So, we must move to Log4j2 since `1.6` :smile: 
Or when the Log4j1 funeral will be...